perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/MembersManagerBlImpl.java

```diff
@@ -906,6 +907,37 @@ public RichMember getRichMemberWithAttributes(PerunSession sess, Member member)
 		return richMembers;
 	}
 
+	// works with RESOURCE and GROUP
```

Comment:
I guess rest of such methods doesn't have proper javadoc either, but it would help to determine, what method does or should do. Please add it for new methods at least.